polynote-runtime/src/main/scala/polynote/runtime/Runtime.scala

```diff
@@ -16,6 +16,9 @@ object Runtime {
     def content(contentType: String, content: String): Unit = displayer(contentType, content)
   }
 
+  // for Python
+  def getDisplay = display
```

Comment:
So `kernel.display` doesn't work in python when `display` is a method of `kernel`? But having a `getX` method makes it work?

Reply:
Curious if that's a jep thing or a python thing?

Reply:
If we change `display` to a method then it would work, though in Python you'd have to do `kernel.display()`, because just `display` would get the method itself without calling it. 

If we could somehow get the `PyObject` that jep creates to have `display` as an attribute then the `kernel.display` notation would work - the question is how to convince jep to do that :)